data/report.txt

```diff
@@ -324,3 +324,10 @@ HawaiiCorrectiveGuided2020 (Timesteps: 24)
 ----------------------------------------------------------------------------
 eo_data_complete    329
 ✔ training amount: 329, positive class: 21.0%
+
+
+
+EthiopiaTigrayGhent2021 (Timesteps: 12)
+----------------------------------------------------------------------------
+eo_data_exporting    161
+✖ validation: 161 labels, but 0 features
```

Comment:
As time passes and these eo data becomes available the 0 features should grow to 161 features